tests/python/common/test_traversal.py

```diff
@@ -19,7 +19,7 @@ def toset(x):
     # F.zerocopy_to_numpy may return a int
     return set(F.zerocopy_to_numpy(x).tolist())
 
-@unittest.skipIf(F.is_hip(), reason="Not implemented in ROCm")
+@unittest.skipIf(F.get_rocm_version() < 7.0, f"Requires ROCm >= 7.0")
```

Comment:
This will skip the test on non-rocm as well


```suggestion
@unittest.skipIf(F.is_hip() and F.get_rocm_version() < 7.0, f"Requires ROCm >= 7.0")
```

Also, FWIW no skip was required in the other port, so I don't think there's something inherent to the rocm version here (https://github.com/nod-ai/dgl/blob/2fc70476f8/tests/python/common/test_traversal.py#L24)